internal/crowdactions/crowdaction_test.go

```diff
@@ -0,0 +1,36 @@
+package crowdaction_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	cwd "github.com/CollActionteam/collaction_backend/internal/crowdactions"
+	m "github.com/CollActionteam/collaction_backend/internal/models"
+	"github.com/CollActionteam/collaction_backend/pkg/mocks/repository"
+
+	"github.com/CollActionteam/collaction_backend/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCrowdaction_GetCrowdactionById(t *testing.T) {
+	as := assert.New(t)
+	dynamoRepository := &repository.Dynamo{}
+	var ctx context.Context
+	var crowdactions *m.CrowdactionData
+	crowdactionID := "sustainability#food#185f66fd"
+
+	t.Run("dev stage", func(t *testing.T) {
+		dynamoRepository.On("GetById", utils.PKCrowdaction, crowdactionID).Return(crowdactions, nil).Once()
+
+		service := cwd.NewCrowdactionService(dynamoRepository)
+
+		crowdaction, err := service.GetCrowdactionById(ctx, crowdactionID)
+
+		fmt.Println("Hello world", crowdaction)
```

Comment:
Can this be removed?

Reply:
We could, but then this would give me an error that the crowdaction variable is not being used. What do we do in this case?

Reply:
> We could, but then this would give me an error that the crowdaction variable is not being used. What do we do in this case?

It depends on your test. If you want to test, that `GetCrowdactionById` logic works, then you should assert that your result has valid field values. If you don't care about crowdaction result, then write:

`_, err := service.GetCrowdactionById(ctx, crowdactionID)`

Reply:
Thanks, that was helpful!